### PR TITLE
feat: add ocm-operator docs

### PR DESCRIPTION
**Pull request by scottd018**

### CODE REVIEW

File: content/docs/rosa/rosa-gitops/_index.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+so that we can get this fixed.
+
+1. Run the script to create the required policies and roles.  This creates a permission boundary, 
+a policy for the operator, and a role which allows the operator to assume a role against the OIDC 
+identity of the ROSA cluster.  If the policies and roles already exist (prefixed by your cluster 
+name), then the creation of them is skipped:
+
+```bash
+curl -s https://raw.githubusercontent.com/rh-mobb/ocm-operator/main/test/scripts/generate-iam.sh | bash
+```
```
Comment (Conan-Kudo) [inline content/docs/rosa/rosa-gitops/_index.md:97]: I'm not sure if I'm super-comfortable with this. Can we do this is a safer way?

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]
Just one thing I'm concerned about, otherwise it looks great!

File: content/docs/rosa/rosa-gitops/_index.md (hunk 1 above)
Comment (scottd018) [reply content/docs/rosa/rosa-gitops/_index.md:97]: The manual steps to do this are far more intrusive and error prone than this.  I am putting together terraform for it now if you prefer to include that.

Comment (Conan-Kudo) [reply content/docs/rosa/rosa-gitops/_index.md:97]: That'd be great, but to unblock this PR, I think it'd be okay with just changing this so we download the file and chmod it. I mostly don't like just automatically passing through instructions to the shell from the internet.

Comment (scottd018) [reply content/docs/rosa/rosa-gitops/_index.md]: @Conan-Kudo Should be fixed now.  Let me know if this works for you:

https://github.com/rh-mobb/documentation/pull/377/commits/f12c0206f838d9de967d367d0c595a669ac9a150

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**